### PR TITLE
[CIAPP-2255] Add CirclecI docs for webhooks for ci visibility

### PR DESCRIPTION
**Pull request by AdrianLC**
### What does this PR do?
Add new page to CI Visibility docs for CircleCI support.

### Preview

https://docs-staging.datadoghq.com/adrian.lopezcalvo%2Fci-visibility-circleci-docs/continuous_integration/setup_pipelines/circleci

### Additional Notes
**To be kept in draft status and not merged until we test in private with some organizations.**

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_pipelines/circleci.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* **Webhook URL**: `https://webhooks-http-intake.logs.datadoghq.com/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
+* **Name**: `Datadog CI Visibility` or any other identifier name that you want to provide.
+* **Events**: Select `Workflow Completed` and `Job Completed`.
+* **Certificate verifications**: Enable this check.
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{< /site-region >}}
+
+{{< site-region region="eu" >}}
+* **Webhook URL**: `https://webhooks-http-intake.logs.datadoghq.com/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
```
Comment (cg14823) [inline content/en/continuous_integration/setup_pipelines/circleci.md:30]:
```suggestion
* **Webhook URL**: `https://webhooks-http-intake.logs.datadoghq.eu/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
```

**Review by cg14823 — CHANGES_REQUESTED** [review]

**Comment by edrevo** [comment]
Preview URL isn't working

**Comment by AdrianLC** [comment]
> Preview URL isn't working

It's working now

**Review by cg14823 — APPROVED** [review]

**Review by edrevo — APPROVED** [review]

File: content/en/continuous_integration/setup_pipelines/circleci.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+{{< /site-region >}}
+
+{{< site-region region="eu" >}}
+* **Webhook URL**: `https://webhooks-http-intake.logs.datadoghq.eu/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
+* **Name**: `Datadog CI Visibility` or any other identifier name that you want to provide.
+* **Events**: Select `Workflow Completed` and `Job Completed`.
+* **Certificate verifications**: Enable this check.
+
+[1]: https://app.datadoghq.eu/account/settings#api
+{{< /site-region >}}
```
Comment (fermayo) [inline content/en/continuous_integration/setup_pipelines/circleci.md:36]:
```suggestion
{{< /site-region >}}

Afterwards, click on **Add Webhook** to save the new webhook.
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**